gradle.properties

```diff
@@ -1,3 +1,4 @@
 android.enableJetifier=true
 android.useAndroidX=true
-org.gradle.jvmargs=-XX:MaxPermSize=1024m -Xmx4096m
\ No newline at end of file
+org.gradle.jvmargs=-XX:MaxPermSize=1024m -Xmx4096m
+kotlin.jvm.target.validation.mode = IGNORE
```

Comment:
I didn't understand why you added kotlin.jvm.target.validation.mode = IGNORE. Could you please explain?